demosplan/DemosPlanCoreBundle/Logic/Import/Statement/StatementFromRowBuilderWithZipSupport.php

```diff
@@ -69,28 +90,48 @@ public function setOriginalFileReferences(Cell $cell): ?ConstraintViolationListI
         return null;
     }
 
-    public function setFileReferences(Cell $cell): ?ConstraintViolationListInterface
+    public function setOriginalFileReferences(?Cell $cell): ?ConstraintViolationListInterface
```

Comment:
Like all other setter methods, there is no reason for this method to allow `null` as parameter.